### PR TITLE
refactor: rename to app-hub

### PR DESCRIPTION
**Pull request by erikarenhill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# App Store for DHIS 2
+# App Hub for DHIS 2
 
-Visit the live [DHIS 2 app store](https://play.dhis2.org/appstore/)
+Visit the live [DHIS 2 app hub](https://play.dhis2.org/apphub/)
```
Comment (varl) [inline README.md]: The URL should be https://apps.dhis2.org.

File: README.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Note: The master branch currently holds the version 2 of the unreleased App Stor
 
 ## Clone the repo
 ```bash
-git clone https://github.com/dhis2/dhis2-appstore.git
+git clone https://github.com/dhis2/dhis2-apphub.git
```
Comment (varl) [inline README.md]: The repo URL is wrong, it should be https://github.com/dhis2/app-hub.git

File: README.md (hunk 1 above)
Comment (varl) [reply README.md]: Resolved but not changed?

Comment (erikarenhill) [reply README.md]: sorry, changed an url the same way on another place... now also updated the readme

**Pull request merged**